USING_IMPLEMENTATION_GUIDES.md

```diff
@@ -0,0 +1,151 @@
+# Using FHIR Implementation Guides
+
+An [Implementation Guide (IG)](https://www.hl7.org/fhir/implementationguide.html) is a set of rules that describe how FHIR resources should be used to solve a particular problem. Using IGs, you can make your FHIR server compliant for country-specific set of rules. IGs can also describe a proper way to represent particular content in FHIR (for example, the breast cancer medical diagnostic process).
+
+FHIR is a platform specification. It describes a general set of capabilities that can be used to solve many healthcare data exchange problems.
+
+IGs are distributed as [packages similar to NPM packages](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification)
+
+## Prerequisites
+
+The prerequisites for FHIR IGs are same as in the FHIR [installation documentation](INSTALL.md). In addition, you need the following: 
+
+1. Java 8 or higher. We recommend using [Amazon Corretto](https://aws.amazon.com/corretto/)
+2. [Maven](https://maven.apache.org/install.html)
+
+## Installation Steps
+
+1. Download the IG packages. IG packages can be downloaded from different sources. The most common sources are the corresponding official IG website (the [US Core website](https://www.hl7.org/fhir/us/core/downloads.html) or the [FHIR Package Registry](https://registry.fhir.org/).
```

Comment:
I think that it's better to add "for example", since US Core is just one of the many IGs that exist. We use it as an example across the docs since it is the most commonly used in the US.

Also, added a link to the specific download per feedback from Tim, https://github.com/awslabs/fhir-works-on-aws-deployment/pull/250#discussion_r594709814

```suggestion
1. Download the IG packages. IG packages can be downloaded from different sources. The most common sources are the corresponding official IG website (for example, [download](https://www.hl7.org/fhir/us/core/package.tgz) from the [US Core website](https://www.hl7.org/fhir/us/core/downloads.html)) or the [FHIR Package Registry](https://registry.fhir.org/).
```

Reply:
Updated.